tests/func-tests/virtual_machines_test.go

```diff
@@ -45,7 +48,16 @@ var _ = Describe("[rfe_id:273][crit:critical][vendor:cnv-qe@redhat.com][level:sy
 
 func verifyVMICreation(client kubecli.KubevirtClient) string {
 	By("Creating VMI...")
-	vmi := kvtests.NewRandomVMI()
+	vmi := libvmi.New(
+		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+		libvmi.WithNetwork(kubevirtcorev1.DefaultPodNetwork()),
+	)
+
+	vmi.ObjectMeta.Namespace = testsuite.GetTestNamespace(vmi)
```

Comment:
please use (if really needed, not so sure about its default) `libvmi.WithNamespace(testsuite.GetTestNamespace(vmi)),`

Reply:
this is tricky. the `vmi` variable is not created yet.

Reply:
OK. now I get it. it should not be set at all. The client later uses NS at creation